Assets/Scripts/LevelController.cs

```diff
@@ -15,20 +15,21 @@ public class LevelController : MonoBehaviour
 
     private void Awake()
     {
-        currentLevel = SceneManager.GetActiveScene();
-        //var instances = GameObject.FindObjectsOfType<LevelController>();
-        //if (instances.Length > 1) Destroy(this.gameObject);
-        //DontDestroyOnLoad(this.gameObject);
+        var instances = GameObject.FindObjectsOfType<LevelController>();
+        if (instances.Length > 1) Destroy(this.gameObject);
+        DontDestroyOnLoad(this.gameObject);
     }
 
     private void Start()
     {
-        //currentLevel = SceneManager.GetActiveScene();
+        SceneManager.sceneLoaded += OnSceneLoaded;
     }
 
-    private void OnEnable()
+    public void OnSceneLoaded(Scene scene, LoadSceneMode mode)
     {
+        currentLevel = SceneManager.GetActiveScene();
         monsters = FindObjectsOfType<Monster>();
+        Debug.Log("monsters: " + monsters.Length);
```

Comment:
Are we keeping `Debug.Log` entries in merged code or was this just for testing?

Reply:
Removed